### PR TITLE
Add tips for writing docs, tweak doc scripts

### PR DESCRIPTION
**Pull request by mway**

### CODE REVIEW

File: docs/misc/writing_docs.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+| :-: | --- |
+| BAD | ~~You will need to set Value X to "foo", as our testing indicated that "bar" can cause issues when solar rays are present.~~ |
+| BAD | ~~We suggest setting Value X to "foo", as testing indicated that "bar" can cause issues when solar rays are present.~~ |
+| GOOD | **Value X must not be set to "bar", as testing indicated that it may cause issues when solar rays are present.** |
+| GOOD | **Value X must be set to "foo"; a value of "bar" may cause issues when solar rays are present.** |
+
+## Tip #2: Avoid subjective language.
+Subjective language – language that is open to interpretation, particularly based on perspective –
+should be avoided wherever possible. Documentation should state objective or broadly empirical
+truths (and should be supported by facts). Introducing perspective inherently creates ambiguity and
```
Comment (benraskin92) [inline docs/misc/writing_docs.md]: No fake news!

File: docs/misc/writing_docs.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+vagueness in the document, as its meaning becomes reader-dependent. Importantly, cases of subjective
+language are distinct from cases of conditional behavior.
+
+| Examples | |
+| :-: | --- |
+| BAD | ~~Setting PowerLevel to 9000 is required, because efficiency is better than reliability in high-rate situations.~~ |
+| GOOD | **In high-rate situations (≥1k req/s), PowerLevel should be set to 9000 in order to maximize efficiency; otherwise, the system defaults to ensuring reliability.** |
+
+## Tip #3: Focus on concrete ideas.
+
```
Comment (benraskin92) [inline docs/misc/writing_docs.md]: Either no line or line immediately after the `Tip`

File: docs/misc/writing_docs.md (hunk 1 above)
Comment (mway) [reply docs/misc/writing_docs.md]: preach

File: docs/misc/writing_docs.md (hunk 2 above)
Comment (mway) [reply docs/misc/writing_docs.md]: 🙏

File: docs/misc/writing_docs.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+can [help with readability][readability] by presenting digestible, unambiguous ideas.
+[Verb tense][verb-tense] and [pronouns][pronouns] can play an important role as well. Ultimately,
+the best litmus test may be the simplest: whether a reader needs to read a sentence twice to
+understand it (technical terms notwithstanding).
+
+Realistically, it is impractical to strive for or expect perfect grammar. The goal isn't to write
+flawless English; the goal is to write _clear, concise, unambiguous, and readable_ documentation,
+for which grammar is an important (albeit not the only) tool.
+
+## Tip 14: Use verb tenses consistently.
```
Comment (benraskin92) [inline docs/misc/writing_docs.md]: Nit: Add `#` before `14`

File: docs/misc/writing_docs.md [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+causes Behavior Y, but will be updated to cause Behavior Z in the future").
+
+## Tip #15: Make consistent references.
+
+When communicating verbally, there is no need to distinguish between represented forms for a given
+artifact, value, method, etc - one can simply say "foo bar" to reference "the member function or
+property `Bar` on type `Foo`", or "foo dot bar" to reference "the YAML key `bar` nested under
+the key `foo`" (as simple examples).
+
+In written documentation, however, these references can either maintain continuity throughout a
```
Comment (andrewmains12) [inline docs/misc/writing_docs.md]: This is a good point! I agree that there's not a consistent style on this overall (sadly), but should we recommend a standard for the purposes of this repo (as opposed to potentially having different styles on different pages)? Maybe not worth it.

Comment (mway) [reply docs/misc/writing_docs.md]: added some more detail around this, let me know what you think

**Pull request merged**